packages/amplify-category-api/src/provider-utils/awscloudformation/cdk-stack-builder/apigw-stack-transform.ts

```diff
@@ -176,77 +190,40 @@ export class ApigwStackTransform {
     );
 
     // Add resources
-    this.resourceName === ADMIN_QUERIES_NAME
-      ? this.resourceTemplateObj.generateAdminQueriesStack(this.resourceName, authResourceName)
-      : this.resourceTemplateObj.generateStackResources(this.resourceName);
-  }
-
-  async applyOverrides() {
-    const backendDir = pathManager.getBackendDirPath();
-    const overrideFilePath = pathManager.getResourceDirectoryPath(undefined, AmplifyCategories.API, this.resourceName);
-    const overrideJSFilePath = path.join(overrideFilePath, 'build', 'override.js');
-
-    const isBuild = await buildOverrideDir(backendDir, overrideFilePath);
-
-    // skip if packageManager or override.ts not found
-    if (isBuild) {
-      let override;
-      try {
-        ({ override } = await import(overrideJSFilePath));
-      } catch {
-        formatter.list(['No override file found', `To override ${this.resourceName} run "amplify override api"`]);
-        override = undefined;
-      }
-
-      if (override && typeof override === 'function') {
-        let overrideCode: string;
-        try {
-          overrideCode = await fs.readFile(overrideJSFilePath, 'utf-8');
-        } catch (error) {
-          formatter.list(['No override file found', `To override ${this.resourceName} run amplify override api`]);
-          return;
-        }
-
-        const sandboxNode = new vm.NodeVM({
-          console: 'inherit',
-          timeout: 5000,
-          sandbox: {},
-          require: {
-            context: 'sandbox',
-            builtin: ['path'],
-            external: true,
-          },
-        });
-
-        await sandboxNode.run(overrideCode, overrideJSFilePath).override(this.resourceTemplateObj as AmplifyApigwResourceStack);
-      }
+    if (this.resourceName === ADMIN_QUERIES_NAME) {
+      this.resourceTemplateObj.generateAdminQueriesStack(this.resourceName, authResourceName);
+    } else {
+      this.resourceTemplateObj.generateStackResources(this.resourceName);
     }
   }
 
-  private async saveBuildFiles() {
+  /**
+   * Save build artifacts
+   */
+  private async saveBuildFiles(): Promise<void> {
     if (this.resourceTemplateObj) {
       this.cfn = JSONUtilities.parse(this.resourceTemplateObj.renderCloudFormationTemplate());
     }
 
     const resourceDirPath = pathManager.getResourceDirectoryPath(undefined, AmplifyCategories.API, this.resourceName);
     fs.ensureDirSync(resourceDirPath);
 
-    const buildDirPath = path.join(resourceDirPath, PathConstants.BuildDirName);
+    const buildDirPath = nodePath.join(resourceDirPath, PathConstants.BuildDirName);
     fs.ensureDirSync(buildDirPath);
 
     stateManager.setResourceParametersJson(undefined, AmplifyCategories.API, this.resourceName, this.cfnInputParams);
 
-    const cfnFilePath = path.resolve(path.join(buildDirPath, `${this.resourceName}-cloudformation-template.json`));
+    const cfnFilePath = nodePath.resolve(nodePath.join(buildDirPath, `${this.resourceName}-CloudFormation-template.json`));
```

Comment:
Has this code been tested on Mac and Linux.

Reply:
This is a find + replace gone wrong, good catch!